### DISCUSSION TITLE
Favorites in Content Window - Show or Hide everything (EG Issue)

### DISCUSSION THREAD

**Discussion by THEAETIK** [TheHerobrine/DarkerNodes#139] [Ideas]
The thing that always bothered me from the content window is how there's no ability to only show / hide the Favorites completely. This is how Epic made their content browser, and it's not changed for the past 3-4 years at least.

Example: https://i.imgur.com/5YdKkIR.png
I can't bring the split any lower than this.

Is this something you think you can "hack" around? I assume it's hard coded and pretty hard to handle. Worth a shot asking I suppose? 

Thanks!

**Comment by hugoattal** [comment]
@THEAETIK Hum, this seems to be hard coded, but I'll check to be sure 😉
